File: hw/tipantiza/u3/hw26TemplateMethodPattern/TemplateMethod_py/controller/beverage_controller.py
from model.tea import Tea
from model.coffee import Coffee
from model.chocolate import Chocolate

class BeverageController:
    def prepare_all_beverages(self):
        tea = Tea()
        coffee = Coffee()
        chocolate = Chocolate()
        
        print("\nMaking tea ...")
        tea.prepare_recipe()
        
        print("\nMaking coffee ...")
        coffee.prepare_recipe()
        
        print("\nMaking chocolate ...")
        chocolate.prepare_recipe()

File: hw/tipantiza/u3/hw26TemplateMethodPattern/TemplateMethod/src/main/java/ec/edu/espe/templatemethod/controller/BeverageController.java

package ec.edu.espe.templatemethod.controller;
import ec.edu.espe.templatemethod.model.Tea;
import ec.edu.espe.templatemethod.model.Coffee;
import ec.edu.espe.templatemethod.model.Chocolate;

/**
 *
 * @author Alexander Tipantiza, The Softwarrios, @ESPE
 */

public class BeverageController {
    public void prepareAllBeverages() {
        Tea tea = new Tea();
        Coffee coffee = new Coffee();
        Chocolate chocolate = new Chocolate();

        System.out.println("\nMaking tea ...");
        tea.prepareRecipe();

        System.out.println("\nMaking coffee ...");
        coffee.prepareRecipe();

        System.out.println("\nMaking chocolate ...");
        chocolate.prepareRecipe();
    }
}